### ISSUE TITLE
Multiple feature specs in a single yaml file

### ISSUE THREAD

**Issue by davidheryanto** [gojek/feast#181]
**Is your feature request related to a problem? Please describe.**
For **organisation** purpose, I think it may be neater if we **support multiple feature specs in a single yaml file**. I don't know if you guys feel the same way i.e. creating features for an entity is similar to creating a table schema?
```sql
# The counterpart to entity_name in Feast
create table customer
(
    # The counterpart to entity_id in Feast
    id  STRING,
	# The counterpart to feature_id in Feast
    total_item_purchased_last_7_days    INT,
    total_item_purchased_last_14_days   INT,
    average_item_purchased_last_7_days  DOUBLE,
    average_item_purchased_last_14_days DOUBLE,
);
```

The use case I'm considering is when people start using Feast for the first time. **How do we expect them to create the feature specs from scratch?** If we expect them to manually create all the feature spec yaml files one by one, I think it can be a bit inconvenient (especially if they have 50 features to start with). They will have to create 50 files and make sure all are correct.

And usually, I think it is **more convenient** if we can see all the features for an entity in one glance vs looking at many files one by one.

*Of course, this may also be more of a UI / discovery issue, i.e. it can be solved with a proper dashboard, e.g. using BigQuery UI to create and update table schemas.*

*Or maybe we should encourage them to use say the Python SDK to create feature specs from scratch. But we need to make sure they can check/ensure the data types for the fetures are correctly inferred/specified*

**Describe the solution you'd like**
I think just support for multiple feature specs in a yaml file will suffice, or another way to group multiple features in one yaml file, like so:
```yaml
# myentity_features.yaml

id: myentity.feature_double_redis
name: feature_double_redis
entity: myentity
owner: bob@example.com
description: test entity.
valueType:  DOUBLE
uri: https://github.com/bob/example
---
id: myentity.feature_int32_redis
name: feature_int32_redis
entity: myentity
owner: bob@example.com
description: test entity.
valueType:  INT32
uri: https://github.com/bob/example
```
or maybe a list of features in one yaml file
```
# myentity_features.yaml

features:
- id: myentity.feature_double_redis
  name: feature_double_redis
  entity: myentity
  owner: bob@example.com
  description: test entity.
  valueType:  DOUBLE
  uri: https://github.com/bob/example
  
- id: myentity.feature_int32_redis
  name: feature_int32_redis
  entity: myentity
  owner: bob@example.com
  description: test entity.
  valueType:  INT32
  uri: https://github.com/bob/example
```

**TLDR: How to ensure smooth experience when "creating" and "discovering" features for an entity in Feast**

**Comment by woop** [comment]
This issue isn't relevant any more due to changes in Feast 0.3.0. Feature sets will make the process a lot more convenient for end users.
